ml/scripts/validate_yunet_b2.py

```
@@ -0,0 +1,257 @@
+    for thr in SWEEP:
+        det_t = YuNetDetector(score_threshold=thr)
+        gt_all, pred_all, ious = [], [], []
+        fp_orig = fp_hard = 0
+        per_fixture = {}
+        for fid in order:
+            dets = det_t.detect(images[fid])
+            gt, preds = to_eval(fixtures[fid]["faces"], dets)
+            res = evaluate(gt, preds)["aggregate"]
+            # per-GT matched IoU for mean matched IoU
+            for gb in fixtures[fid]["faces"]:
+                bi, bp = best_iou_pred(gb, preds)
```

Ruff (0.16.3)

[warning] Unpacked variable `bp` is never used

(RUF059)